src/main/java/lv/ctco/javaschool/goal/entity/GoalDto.java

```diff
@@ -0,0 +1,70 @@
+package lv.ctco.javaschool.goal.entity;
+
+
+import java.util.List;
+
+public class GoalDto {
```

Comment:
Suggest placing entities (`@Entity`) in one package, and DTOs in another.

Reply:
Done